mail_show_follower/views/res_config_settings.xml

```diff
@@ -10,12 +10,70 @@
+                                        class="o_light_label"
+                                        string="Show Internal Users on CC"
+                                    />
+                                    <field name="show_internal_users_cc" />
+                                </div>
+                                <div>
+                                    <label
+                                        for="show_followers_message_sent_to"
+                                        class="o_light_label"
+                                        style="vertical-align: top;"
+                                    />
+                                    <field
+                                        name="show_followers_message_sent_to"
+                                        placeholder="This message has been sent to"
+                                        style="display:inline-block;"
+                                    />
+                                </div>
+                                <div>
+                                    <label
+                                        for="show_followers_partner_format"
+                                        class="o_light_label"
+                                        style="vertical-align: top;"
+                                    />
+                                    <field
+                                        name="show_followers_partner_format"
+                                        placeholder="%%(partner_name)s &amp;lt;%%(partner_email)s&amp;gt;"
+                                        style="display:inline-block;"
+                                    />
+                                </div>
+                                <div>
+                                    <label
+                                        for="show_followers_message_response_warning"
+                                        class="o_light_label"
+                                        style="vertical-align: top;"
+                                    />
+                                    <field
+                                        name="show_followers_message_response_warning"
+                                        placeholder="Notice: Replies to this email will be sent to all recipients."
+                                        style="display:inline-block;"
+                                    />
+                                </div>
+                            </div>
+                            <label
+                                for="show_followers_message_preview"
+                                class="text-muted"
+                                groups="base.group_no_one"
+                            />
+                            <field
+                                name="show_followers_message_preview"
+                                style="width:100%;"
+                                widget="html"
+                                groups="base.group_no_one"
```

Comment:
I would let users see this field even if they're not in debug mode. It's a very nice visual hint that helps always IMHO. 👍🏼 

```suggestion
```